.github/workflows/build.yml

```diff
@@ -0,0 +1,21 @@
+name: Build and Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build Docker image
+      run: |
+        docker build --build-arg PYTHON_VERSION=${{ matrix.python-version }} -t your-app-name:${{ matrix.python-version }} .
```

Comment:
I think `your-app-name` is a placeholder that should probably be replaced with `piqture`